package.json

```diff
@@ -28,8 +28,14 @@
     "*.js"
   ],
   "dependencies": {
+    "envelope-js": "^1.3.2",
+    "envelope-spec": "^1.1.0",
+    "private-group-spec": "^1.1.3",
     "pull-stream": "^3.6.14",
-    "ssb-meta-feeds": "~0.30.0"
+    "ssb-bfe": "^3.6.0",
+    "ssb-crut": "^4.6.0",
```

Comment:
@Powersource Perhaps an easy task for you would be to update ssb-crut's license from AGPL-3.0 to LGPL-3.0-only? It's something we all already agreed on. https://gitlab.com/ahau/lib/ssb-crut

Reply:
Done, waiting on (i guess mix?) merge and new version published

Reply:
Need to publish

Reply:
This can be done after merging this PR.